tests/e2e/react-router-user-journeys.spec.ts

```diff
@@ -0,0 +1,247 @@
+import { expect, test } from './support/fixtures'
+
+test.describe('React Router user journeys', () => {
```

Comment:
These are almost identical to the Remix e2e tests. We only changed a few strings like "Welcome to Remix", etc.